osbuild/buildroot.py

```diff
@@ -255,11 +256,18 @@ def run(self, argv, monitor, timeout=None, binds=None, readonly_binds=None, extr
         # In case `libdir` contains the python module, it must be self-contained
         # and we provide nothing else. Otherwise, we additionally look for
         # the installed `osbuild` module and bind-mount it as well.
-        mounts += ["--ro-bind", f"{self._libdir}", "/run/osbuild/lib"]
-        if not os.listdir(os.path.join(self._libdir, "osbuild")):
+        for host_dir, target_dir in zip(self._libdir.dirs, self._libdir.buildroot_dirs):
+            mounts += ["--ro-bind", host_dir, target_dir]
+
+        def _find_osbuild_dir():
+            for host_libdir in self._libdir.dirs:
+                if os.path.isdir(os.path.join(host_libdir, "osbuild")):
+                    return True
+            return False
+        if not _find_osbuild_dir():
             modorigin = importlib.util.find_spec("osbuild").origin
             modpath = os.path.dirname(modorigin)
-            mounts += ["--ro-bind", f"{modpath}", "/run/osbuild/lib/osbuild"]
+            mounts += ["--ro-bind", f"{modpath}", f"{self._libdir.buildroot_dirs[0]}/osbuild"]
```

Comment:
Not your change but let's use this opportunity: `f"{modpath}"` need not be an f-string, it's already a `str` by itself.

Let's also also change the f-string for path concatenation to use `os.path.join`?